ab-testing/frontend/src/app.d.ts

```diff
@@ -0,0 +1,13 @@
+// See https://svelte.dev/docs/kit/types#app.d.ts
+// for information about these interfaces
+declare global {
+	namespace App {
```

Comment:
Is this file necessary? Is this a placeholder for future types/interfaces?

Reply:
This is part of the Sveltekit boilerplate, I believe it is required but will be experimenting with removing more of this as I finalise the build process